main.js

```diff
@@ -318,9 +324,11 @@ function createStates(onConnect) {
         },
         connected: (handler) => {
             states = handler;
-            if (states.clearAllLogs)     states.clearAllLogs();
-            if (states.clearAllMessages) states.clearAllMessages();
-            deleteAllZipPackages();
+            if (!compactGroupController) {
```

Comment:
Here it is more clear, but these fine things deserve a comment IMO. Something like "logs are handled by the main controller".

Reply:
Done